nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -578,19 +602,30 @@ void NNTrainer::NNTrainerTrain::createModel() {
   }
   try {
 
-    model->load(model_config,
-                ml::train::ModelFormat::MODEL_FORMAT_INI_WITH_BIN);
+    model->load(model_config, ml::train::ModelFormat::MODEL_FORMAT_INI);
   } catch (const std::exception &e) {
     ml_loge("Error %s, %s", typeid(e).name(), e.what());
     return;
   }
+
   try {
     model->compile();
     model->initialize();
   } catch (const std::exception &e) {
     ml_loge("Error %s, %s", typeid(e).name(), e.what());
     return;
   }
+
+  try {
+    if (!model_load_path.empty()) {
+      ml_loge("load path : %s", model_load_path.c_str());
+      model->load(model_load_path, ml::train::ModelFormat::MODEL_FORMAT_BIN);
```

Comment:
Does the intended flow : load configure(ini file) first and load weight if there is bin file ?

Reply:
Yes.